### PR TITLE
Updated MR Dev Center links

### PR DESCRIPTION
**Pull request by rwinj**
Overview
---

Changes
---
- Fixes: # .

### CODE REVIEW

**Comment by wiwei** [comment]
Our github.io docs are generated off of mrtk_development, so I'm retargetting to mrtk_development. 

It's a bit weird, but we're working through fixing this up now :)

**Comment by rwinj** [comment]
Right. I got that from Bernadette too. I made the changes in mrtk_development as well. Maybe disregard this one.

________________________________

File: README.md [hunk 1]
```diff
@@ -85,10 +85,16 @@ For issues related to Windows Mixed Reality that aren't directly related to the
...
-| See code examples. Do a coding tutorial. Watch guest lectures.          | Get design guides. Build user interface. Learn interactions and input.     | Get development guides. Learn the technology. Understand the science.       | Join open source projects. Ask questions on forums. Attend events and meetups. |
+| Learn to build mixed reality experiences for HoloLens and immersive headsets (VR).          | Get design guides. Build user interface. Learn interactions and input.     | Get development guides. Learn the technology. Understand the science.       | Get your app ready for others and consider creating a 3D launcher. |
+
+# Useful resouces on Azure
+| ![Azure Spatial Anchors](External/ReadMeImages/icon_academy.png) [Azure Spatial Anchors](https://docs.microsoft.com/en-us/azure/spatial-anchors/)| ![Speech Services](External/ReadMeImages/icon_design.png) [Speech Services](https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/)| ![Vision Services](External/ReadMeImages/icon_development.png) [Vision Services](https://docs.microsoft.com/en-us/azure/cognitive-services/computer-vision/)|
```
Comment (wiwei) [inline README.md]: Do you know if it's possible to add a line break or something to the caption? it looks like this on chrome:

image

**Comment by rwinj** [comment]
I made the changes in ‘mrtk_development’

File: README.md (hunk 1 above)
Comment (rwinj) [reply README.md]: I shortened it to 'Spatial Anchors' - hopefully that fixes it.

**Pull request closed without merging**